.drone.yml

```diff
@@ -1,103 +1,79 @@
 
   vic-ui:
-    image: gcr.io/eminent-nation-87317/vic-integration-test:1.36
+    image: 'gcr.io/eminent-nation-87317/vic-integration-test:1.36'
     pull: true
     environment:
-      BUILD_NUMBER: ${DRONE_BUILD_NUMBER}
       BIN: bin
-      GS_PROJECT_ID: ${GS_PROJECT_ID}
-      GS_CLIENT_EMAIL: ${GS_CLIENT_EMAIL}
-      GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
+    secrets:
+      - gs_client_email
+      - gs_private_key
+      - gs_project_id
     commands:
-      - make vic-ui-plugins
+      - 'export BUILD_NUMBER=${DRONE_BUILD_NUMBER}'
+      - 'make vic-ui-plugins'
     when:
       repo: vmware/vic-ui
-      branch: [ master, releases/*, refs/tags/* ]
+      branch: [master, 'releases/*', 'refs/tags/*']
 
   bundle:
-    image: gcr.io/eminent-nation-87317/vic-integration-test:1.36
+    image: 'gcr.io/eminent-nation-87317/vic-integration-test:1.36'
     pull: true
     environment:
       BIN: bin
       GOPATH: /go
       SHELL: /bin/bash
     commands:
-      - mkdir -p $BIN/ui
-      - tar -czvf $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz $BIN/ui
-      - mkdir bundle
-      - mkdir bundle-release
-      - cp $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz bundle
-      - cp $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz bundle-release/vic_ui_`git describe --tags $(git rev-list --tags --max-count=1)`.tar.gz
-      - rm -rf $BIN
-      - ls -la bundle
+      - 'mkdir -p $BIN/ui'
+      - 'tar -czvf $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz $BIN/ui'
+      - 'mkdir bundle'
+      - 'mkdir bundle-release'
+      - 'cp $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz bundle'
+      - 'cp $BIN/vic_ui_${DRONE_BUILD_NUMBER}.tar.gz bundle-release/vic_ui_`git describe --tags $(git rev-list --tags --max-count=1)`.tar.gz'
+      - 'rm -rf $BIN'
+      - 'ls -la bundle'
     when:
       repo: vmware/vic-ui
-      event: [ push, tag ]
-      branch: [ master, releases/*, refs/tags/* ]
+      event: [push, tag]
+      branch: [master, 'releases/*', 'refs/tags/*']
 
   publish-gcs-builds-on-pass:
-    image: maplain/drone-gcs:latest
+    image: 'maplain/drone-gcs:latest'
     pull: true
     source: bundle
     target: vic-ui-builds
     acl:
-      - allUsers:READER
-    cache_control: public,max-age=3600
+      - 'allUsers:READER'
+    cache_control: 'public,max-age=3600'
     when:
       repo: vmware/vic-ui
-      event: [ push ]
-      branch: [ master, releases/* ]
+      event: [push]
+      branch: [master, 'releases/*']
       status: success
 
   publish-gcs-releases:
-    image: maplain/drone-gcs:latest
+    image: 'maplain/drone-gcs:latest'
     pull: true
     source: bundle-release
     target: vic-ui-releases
     acl:
-      - allUsers:READER
-    cache_control: public,max-age=3600
+      - 'allUsers:READER'
+    cache_control: 'public,max-age=3600'
     when:
       repo: vmware/vic-ui
-      event: [ push, tag ]
-      branch: [ refs/tags/* ]
+      event: [push, tag]
+      branch: ['refs/tags/*']
       status: success
 
-  report-coverage:
```

Comment:
Why is the code coverage section removed?

Reply:
That seems like a mistake, I will add it back.